### ISSUE TITLE
Error installing due to eslint-plugin-jsx-a11y

### ISSUE THREAD

**Issue by FreddieShoreditch** [relax/relax#230]
As above:

On `npm install` on a blank machine I get:

```bash
npm WARN eslint-config-airbnb@7.0.0 requires a peer of eslint-plugin-jsx-a11y@^0.6.2 but none was installed.
```

**Comment by MervynFang** [comment]
the same

**Comment by bruno12mota** [comment]
I believe this is fixed from 8d94e4b82561ad9e1b5ca01257204deec39fc587 thanks for reporting 👍

**Comment by elzii** [comment]
I was able to resolve using these steps:

First manually install eslint-plugin-jsx-a11y version:

`npm install eslint-plugin-jsx-a11y@^0.6.2`

Then, another error is thrown:

```
ERROR: Found libvips 8.0.2 but require 8.2.0
Please upgrade libvips by running: brew update && brew upgrade
```

so run:

`brew update && brew upgrade`

troubleshooting for this step will vary by user, brew is very helpful however.

then running 
`npm install`

again in the project dir is succesful.
